### PR TITLE
Feature/prometheus

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: app/lib/esClient.js
Comment (st3v3nhunt) [inline app/lib/esClient.js]: This won't be stopped if there is an error. Needs fixing.

**Review by markysoft — DISMISSED** [review]
All looks good to me, will wait for Adam's review before merging.

File: app.js
Comment (neilbmclaughlin) [inline app.js:3]: Just an observation - the code uses a mix of styles. I prefer the former.
```
const applicationStarts = require('./app/lib/promCounters').applicationStarts;
[snip]
applicationStarts.inc(1);
```
vs
```
const counters = require('../lib/promCounters');
[snip]
counters.notFound.inc(1);
```

Comment (st3v3nhunt) [reply app.js:3]: I do and that is why its in. The latter is there due to there being multiple counters used in that file.

**Review by adamgraves-choices — DISMISSED** [review]

**Comment by adamgraves-choices** [comment]
@st3v3nhunt Is it possible to have `prometheus.monitoring` = `true` added as a label please? :)

**Comment by st3v3nhunt** [comment]
@adamgraves-choices Label added

**Review by adamgraves-choices — APPROVED** [review]
LGTM.

**Pull request merged**
